airflow/jobs.py

```diff
@@ -665,8 +665,7 @@ def manage_slas(self, dag, session=None):
         slas = (
             session
             .query(SlaMiss)
-            .filter(SlaMiss.notification_sent == False)  # noqa: E712
-            .filter(SlaMiss.dag_id == dag.dag_id)
+            .filter(SlaMiss.notification_sent == False, SlaMiss.dag_id == dag.dag_id)  # noqa: E712
```

Comment:
This change can bring minor performance improvement. 

Benchmarking for this sort of change was done in https://github.com/apache/airflow/pull/4433

Reply:
Oh wow that's mental. I would not have expected that much of a difference! What DB did you test on btw?

Reply:
In the benchmarking in https://github.com/apache/airflow/pull/4433 you can find that:

- `.filter(DM.is_active == True)` and `.filter(DM.is_active)` are very different in terms of performance. I think the main difference is in Python code side rather than DB side (extra calculation of ` == True`).
- `.filter(cond_1).filter(cond_2)` and `.filter(cond_1, cond_2)` are very close. So this change here in this PR is not really brining that big improvement.

I used both _MariaDB_ and _SQLite_ for the benchmarking in https://github.com/apache/airflow/pull/4433 (not sure which one it is for the figures shared. But I remember the results from two DBs are consistent)